### PR TITLE
[TG-145][fix] Dashboard empty view (help text)

### PR DESCRIPTION
**Pull request by demakoff**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/34" target="_blank">ReviewNinja

File: src/app/dashboard/dashboard.test.js [hunk 1]
```diff
@@ -39,4 +39,20 @@ describe('Dashboard', () => {
             expect(sut.executors).toEqual([executor]);
         });
     });
+
+    describe('isDashboardEmpty', () => {
+        it('should return \'true\' if there is no sensors and executors', () => {
```
Comment (disist) [inline src/app/dashboard/dashboard.test.js]: May be better name of the test is 
'should dashboard be is empty when there is no sensors and executors'

to  exclude tech details and call our tests according business logic

File: src/app/dashboard/dashboard.test.js [hunk 2]
```diff
@@ -39,4 +39,20 @@ describe('Dashboard', () => {
...
+            expect(sut.isDashboardEmpty()).toBe(true);
+        });
+
+        it('should return \'false\' if there are sensors', () => {
```
Comment (disist) [inline src/app/dashboard/dashboard.test.js]: also

File: src/app/dashboard/dashboard.test.js [hunk 3]
```diff
@@ -39,4 +39,20 @@ describe('Dashboard', () => {
...
+            sut.sensors = [{}];
+            expect(sut.isDashboardEmpty()).toBe(false);
+        });
+
+        it('should return \'false\' if there are executors', () => {
```
Comment (disist) [inline src/app/dashboard/dashboard.test.js]: also

File: src/app/dashboard/dashboard.test.js (hunk 1 above)
Comment (demakoff) [inline src/app/dashboard/dashboard.test.js]: Ok, agree. But I think the better message will be 'should show empty dashboard if there is no sensors and executors'

**Pull request merged**